src/index.ts

```diff
@@ -44,285 +45,294 @@ interface StorageMetadata extends Record<string, string> {
 	tokenKeyID: string;
 }
 
-export const handleTokenRequest = async (ctx: Context, request: Request) => {
-	ctx.metrics.issuanceRequestTotal.inc({ version: ctx.env.VERSION_METADATA.id ?? RELEASE });
-	const contentType = request.headers.get('content-type');
-	if (!contentType || contentType !== MediaType.PRIVATE_TOKEN_REQUEST) {
-		throw new HeaderNotDefinedError(`"Content-Type" must be "${MediaType.PRIVATE_TOKEN_REQUEST}"`);
-	}
-
-	const buffer = await request.arrayBuffer();
-	const tokenRequest = TokenRequest.deserialize(new Uint8Array(buffer));
-
-	if (tokenRequest.tokenType !== TOKEN_TYPES.BLIND_RSA.value) {
-		throw new InvalidTokenTypeError();
-	}
+// If we move the rotation logic to a separate worker, we can define a rotation-specific named entry point.
+// https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/rpc/
+export class IssuerHandler extends WorkerEntrypoint<Bindings> {
```

Comment:
from a review perspective, it could be worth keeping the methods unchanged (not moving them to a new class)

and declare the new class `Issuer` below which would proxy method, possibly with different method signatures. for instance, I could imagine `handleTokenRequest` to be have the following signature `issue(request: TokenRequest)`

ctx is also specific to this worker, that is hard to provide from another worker.